### ISSUE TITLE
Error at initialization with DnD System 2.1.4

### ISSUE THREAD

**Issue by vt-tom** [mhilbrunner/foundryvtt-dnd5e-lang-de#3]
Error for hook init

Foundry Version 10.291
DnD System 2.1.4
DnD German
Babele 2.3.4

Screenshot:
![grafik](https://user-images.githubusercontent.com/80099175/218419179-de16a884-e280-491f-b473-3fbe640d4ca1.png)

**Comment by glont** [comment]
FYI @mhilbrunner:

It seems in dnd5e version 2.1.x the classFeatures key in CONFIG.DND5E does not exist anymore. As far as I can tell this was deprecated in 1.6.0 in favor for the advancement system and got removed in 2.1, specifically in [this commit](https://github.com/foundryvtt/dnd5e/commit/02f117a328d117a480f9f1048656191ec5f3a4c7#diff-5ab7cd05ad5f0b1b20878ec2638ebb732cb4466abc03963a02fc1c1aba759dc9).

Currently the code in init tries to override things for that value, which creates this error. It is the last thing happening in init, which may mean that everything happens to work, but it is not great. See [dnd5e-de.js#L33-L46)](https://github.com/mhilbrunner/foundryvtt-dnd5e-lang-de/blob/15a16d9a60851a1e58be3a38c05905f31d82a112/dnd5e-de.js#L33-L46) of the current commit to master.

I think those lines can be removed, though I did not test it.
